macros/src/main/scala/gestalt/macros/Optional.scala

```diff
@@ -28,11 +28,14 @@ final class Optional[+A >: Null](val value: A) extends AnyVal {
         Select(tempIdent, "value")
     }
 
-    q"""
+    {
+      import scala.gestalt.options.unsafe
```

Comment:
We can add `package macros` at the head of the file, then change `new Optional(..)` to `new _root_.macros.Optional(...)`.